src/Native/Runtime/EHHelpers.cpp

```diff
@@ -29,31 +29,34 @@
 // address may be to another managed function, or it may be to an unmanaged function, or it may be to a GC
 // hijack. The address may also refer to an EEType if we've been called from RhpGetClasslibFunction. If it is
 // a GC hijack, we will recgonize that and use the real return address, updating the address passed in.
-static Module * FindModuleRespectingReturnAddressHijacks(void * address)
+static ICodeManager * FindCodeManagerRespectingReturnAddressHijacks(void * address)
```

Comment:
The comment is obsolete - it says "Find the module".
Also, recgonize -> recognize